caddy/caddymain/run.go

```diff
@@ -97,6 +97,12 @@ func Run() {
 		mustLogFatalf(err.Error())
 	}
 
+	// Execute Startup Hooks
```

Comment:
A more useful comment might be, "Execute plugins that are registered to run as the process starts"